main.py

```diff
@@ -167,6 +168,12 @@ def campaign_group_page(campaign_group=None, pwg=None):
     conn = sqlite3.connect('data.db')
     cursor = conn.cursor()
     sql_args = [campaign_group]
+    counts = cursor.execute("SELECT COUNT(*) FROM samples WHERE campaign_group = ?", sql_args)
```

Comment:
This is missing `WHERE` for interested PWG, like lines 210-211 for `sql_query` and `sql_query_ul`:
```
sql_query += ' AND interested_pwgs LIKE ?'
sql_query_ul += ' AND resp_group LIKE ?'
```